css/main.css

```diff
@@ -0,0 +1,165 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Outfit';
+  sans-serif;
+  background: #0d1a2d;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+section {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.line {
+  background: #15273f;
+  box-shadow: 0 0 0 12px hsl(217deg 55% 10%), 0 0 0 32px hsl(216deg 57% 11%);
+  border-radius: 10px;
+  width: 300px;
+  height: 500px;
+}
+
+.card {
+  margin: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card .main-image-container {
+  position: relative;
+  width: 280px;
+  height: 280px;
+}
+
+.card .main-image-container .equ-container {
+  width: 100%;
+  height: 100%;
+}
+
+.card .main-image-container .equ-container>img {
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.card .main-image-container .overlay-image-container {
+  border-radius: 10px;
+  opacity: 0;
+  cursor: pointer;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #00ffff9c;
+}
+
+.main-information {
+  line-height: 1.6rem;
+}
+
+.main-h {
+  display: flex;
+  justify-content: space-between;
+  margin: 10px 0;
+}
+
+.primary-p {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.secondary-p {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 10px 0;
+}
+
+.p1 {
+  display: flex;
+  justify-content: space-evenly;
+  column-gap: 10px;
+  color: #00ffff !important;
+}
+
+.eth {
+  width: 12px;
+  height: 24px;
+}
+
+.clock {
+  width: 18px;
+  height: 24px;
+}
+
+.p2 {
+  display: flex;
+  justify-content: space-evenly;
+  column-gap: 10px;
+}
+
+.creator {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  column-gap: 10px;
+  border-top: grey solid 1px;
+  margin: 15px 0;
```

Comment:
You can remove the margin completely and just add padding-top to add some space between the elements and line